src/helpers/fallbackAuthHelper.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global console, location, Office, Microsoft */
+
+import { MSGraphHelper } from "./../../node_modules/office-addin-sso/lib/msgraph-helper";
```

Comment:
office-addin-sso hasn't been published to NPM yet so we can't include it in project dependencies in package.json. Once it's published, it will be included (and linking to office-addin-sso) will no longer be required

Reply:
OK. But why are these being treated as libraries at all? They fall far short of being general purpose graph or odata libraries, to put it mildly.  The Graph team has created SDKs for several languages. Unfortunately, there isn't one for Typescript and the one for JavaScript can only be used with Implicit Flow (and is essentially unusable in an Office Add-in for other reasons). But the responsibility should be on that team to create a genuine TS and JS server-side SDK, shouldn't it?

Reply:
We made it to abstract the information from the developer so they can get started quickly, and to be able to push new updates to them as the SSO APIs and process solidifies. Once SSO GAs, I think it is fair to say that we may no longer need this. But until then, I think there is still need. Thoughts?

Reply:
@nokafor I'm actually _more_ confused now. The msgraph- and odata- helpers have no intrinsic connection with SSO. They end up being called by the fallback path as well as the SSO path, which means they could be used in an add-in that doesn't use SSO at all. We would not be updating them "as the SSO APIs and process solidifies". 
Also, how does treating them as libraries in /node_modules as distinct from source files in the template help devs get started any quicker? Either way, the code is provided for them.

Reply:
I feel like we are having a broad philosophical debate here. Having this stuff abstracted from the user quite simply reduces the complexity of the project for the user. If you are merely concerned about having the ms-graphhelper and odata-helper grouped under the name "office-addin-sso", we can call the package something else (please suggest a name if that's the case)

Reply:
I already did, either in another conversation in this PR or the PR for the JS template. My suggestion was "office-add-in-helpers". We can revisit whether these should be libraries at all in the future.

Reply:
That's too generic, as essentially everything under Office-Addin-Scripts is a helper. The package currently contains a config portion, which is entirely specific to SSO, a server component, which is also specific to SSO.

Reply:
Agreed, that is too generic. If this is for SSO, then it should go in the office-addin-sso package.

Reply:
I didn't realize that there were other things in the package. If some of them are entirely specific to SSO, then go ahead and include sso in the package name. But since these two files are NOT sso-specific, ideally they would go into a separate package ("office-add-in-msgraph"?). (Or no package at all.)